sdk/rpc/proto/protocol.go

```diff
@@ -76,23 +76,43 @@ type ImportCredentialRequest struct {
 	sdk.ImportOutput
 }
 
+type ImportCredentialResponse struct {
+	*sdk.ImportOutput
```

Comment:
The code might become a little easier if we drop this pointer. It's no longer needed if the methods of `RPCServer` start accepting a pointer value as their second argument (see other comment).

Reply:
Done!

Reply:
It does not strictly _have_ to be a pointer, but it does more easily allow us to check if there was a panic (by checking if `Panic` is `nil`).

Alternatively, we can also add an extra field that indicates whether the call was successful or not, but a pointer should work just fine.

Reply:
This makes sense, thanks!